python-package/setup.py

```diff
@@ -28,6 +28,7 @@
       install_requires=[
           'numpy',
           'scipy',
+          'scikit-learn',
```

Comment:
another option can be, in the regular `setup.py` we don't ask for `scikit-learn` and keep the dependency on `pip` installation for `pip` users?